src/repository/Repository.ts

```diff
@@ -12,7 +12,7 @@ import {SelectQueryBuilder} from "../query-builder/SelectQueryBuilder";
 /**
  * Repository is supposed to work with your entity objects. Find entities, insert, update, delete, etc.
  */
-export class Repository<Entity extends ObjectLiteral> {
+export class Repository<Entity extends ObjectLiteral> implements IRepository<Entity> {
```

Comment:
why do you need it? Why you simply don't do `implements Repository` where `Repository` is a class? (in typescript you can implement classes as well)

Reply:
@pleerock you just made my life a little bit easier.

Cheers!